app/searchApi.rb

```diff
@@ -204,20 +204,20 @@ def get_query_display(params)
   end
     
   display_params = {
-    'q' => params['q'] ? params['q'].join(" ") : 'test',
-    'rows' => params['rows'].length > 0 ? params['rows'][0] : '10',
-    'sort' => params['sort'].length > 0 ? params['sort'][0] : 'rel',
-    'start' => params['start'].length > 0 ? params['start'][0] : '0',
+    'q' => params['q'] ? params['q'].join(" ") : '',
+    'rows' => params.dig('rows', 0) ? params['rows'][0] : '10',
+    'sort' => params.dig('sort', 0) ? params['sort'][0] : 'rel',
+    'start' => params.dig('start', 0) ? params['start'][0] : '0',
```

Comment:
For future reference an even more compact way would, I think be:
`'start' => params.dig('start', 0) || '0'`